welcome-checklist.md

```diff
@@ -26,11 +26,18 @@ Before we begin, we ask that you complete this list to ensure that you have acce
 
 * In the terminal, type `python --version`. You should see the version of your Python program listed.
     * Mac and Linux: If you get an error or your version does not indicate "Anaconda", put your red sticky note on top of your laptop.
-    * Windows: If you receive an error message about python not being found, run the following command:
-      ```
-      echo  ". \$USERPROFILE/AppData/Local/Continuum/anaconda3/Scripts/activate" >> ~/.bashrc
-      ```
-      then open a **new** terminal and try typing `python --version` again.
+    * Windows: If you receive an error message about python not being found, do the following:
+      * Open the Anaconda Prompt and enter the following to find the path of the Anaconda3 directory:
+        ```
+        where python
+        ```
+        The output should show a path similar to `C:\Users\Username\Anaconda3\python.exe`.
+      * Convert that path by lower-casing the drive letter and removing the colon, replacing all instances of `\` with `/` including one at the beginning, and removing the last `\python.exe` part. So, for example, if you got `C:\Users\Username\Anaconda3\python.exe` from the previous step, you will use `/c/Users/Username/Anaconda3` for the next step.
```

Comment:
I wonder if it would be more clear to present this more like: 
```
* Convert that path by doing the following:
 * Replace all instances of `\` with `/`
 * Replace `C:` with `/c`
 * Remove the`\python.exe` part at the end.
```

Reply:
This is why I didn't become an English major. I say simple things in a complicated way. Or maybe I _should_ have become an English major..... Anyways, I like your suggestion, gonna use it.